### ISSUE TITLE
How to read multi parquets files from S3. 

### ISSUE THREAD

**Issue by ghost** [modin-project/modin#700]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: macOS 10.14.5 (18F132)
- **Modin installed from (source or binary)**: pip install 
- **Modin version**: 0.5.3 
- **Python version**: Python 3.7.3 
- **Exact command to reproduce**:

### Describe the problem
I have bunch of parquets files store in S3. 
I want to load all the files into one dataframe in distributed way. 
How can I do it?

**Comment by devin-petersohn** [comment]
Hi @oshriPP thanks for the question!

This functionality is not supported in pandas or Modin, although we could add a way to do it from `modin.experimental.pandas`, which has additional functionality augmenting pandas. 

I will add it to our list of projects in #658. Thanks again for the question!

**Comment by FelixGSE** [comment]
Is this still to be considered? That would be super helpful

**Comment by devin-petersohn** [comment]
Closing this issue as this feature has been implemented since this issue was opened.

@FelixGSE I saw your comment on the Modin Slack, it is supported to read the data in parallel from a directory. Your issue should not be related to the support of the feature, but something else seems to be happening. Let's continue in a new issue or on the Modin slack to get this resolved if that is okay with you.
